suse_ha-formula/suse_ha/sbd.sls

```diff
@@ -73,7 +73,7 @@ sbd_sysconfig:
   suse_sysconfig.sysconfig:
     - name: sbd
     - uncomment: '#'
-    - quote: False
+    - quote_char: "'"
```

Comment:
Agreed in general, but I wonder if you really need to specify the quote_char - the default `"` should work.

Reply:
To avoid expansion of potential variables.

Reply:
I'd call variable expansion a feature of sysconfig files (even if it's rarely used) - but if you think in this case it makes sense to prevent variable expansion *), feel free to merge as is.

*) I can't really judge since I'm not familiar with SBD

Reply:
Fair, but since apparently already semicolons cause issues in the script sourcing it, I figured I rather not have it interpret _anything_ other than strings. ;)